### PR TITLE
Custom KeyTool implementation

### PR DESCRIPTION
**Pull request by kpudlik**

### CODE REVIEW

File: src/main/scala/org/constellation/crypto/KeyStoreUtils.scala [hunk 1]
```diff
@@ -83,7 +83,7 @@ object KeyStoreUtils {
     alias: String,
     storePassword: Array[Char],
     keyPassword: Array[Char]
-  ): F[KeyStore] =
+  ): F[Either[Throwable, KeyStore]] =
```
Comment (marcinwadon) [inline src/main/scala/org/constellation/crypto/KeyStoreUtils.scala]: Why not EitherT?

**Comment by marcinwadon** [comment]
For such a small thing there is no much sense to keep everything polymorphic but please keep in mind that ideally all those functions should be “IO-agnostic” so using tagless final pattern and passing the proper type only once, at the end of the world (run).

**Comment by kpudlik** [comment]
> For such a small thing there is no much sense to keep everything polymorphic but please keep in mind that ideally all those functions should be “IO-agnostic” so using tagless final pattern and passing the proper type only once, at the end of the world (run).

Yep, I was wondering if it is worth to do it in separate jar and such small code. Fixed anyway.

File: src/main/scala/org/constellation/crypto/KeyStoreUtils.scala (hunk 1 above)
Comment (kpudlik) [reply src/main/scala/org/constellation/crypto/KeyStoreUtils.scala]: Just a leftover, I accidentally moved EitherT conversion to other place. On it.

**Pull request closed without merging**